test/TesterAzureUtils/SiloInstanceTableManagerTests.cs

```diff
@@ -22,25 +24,23 @@ public class SiloInstanceTableManagerTests : IClassFixture<SiloInstanceTableMana
     {
         public class Fixture
         {
-            public Fixture()
-            {
-                LogManager.Initialize(new NodeConfiguration());
-            }
         }
 
         private string deploymentId;
         private int generation;
         private SiloAddress siloAddress;
         private SiloInstanceTableEntry myEntry;
         private OrleansSiloInstanceManager manager;
-        private readonly Logger logger;
+        private readonly ILogger logger;
         private readonly ITestOutputHelper output;
 
         public SiloInstanceTableManagerTests(ITestOutputHelper output)
         {
             TestUtils.CheckForAzureStorage();
             this.output = output;
-            logger = LogManager.GetLogger("SiloInstanceTableManagerTests", LoggerType.Application);
+            var loggerFactory = TestingUtils.CreateDefaultLoggerFactory(new NodeConfiguration().TraceFileName);
+                
+            logger = loggerFactory.CreateLogger<SiloInstanceTableManagerTests>();
```

Comment:
please create (and then dispose) the logger factory in the Fixture itself (and make it accessible as a property, so that it can be injected in the test class' constructor)

Reply:
I switched to use output.WriteLine